app/assets/v2/js/pages/fulfill_bounty.js

```diff
@@ -16,161 +16,164 @@ window.onload = function() {
 
   // a little time for web3 injection
   setTimeout(function() {
-    waitforWeb3(actions_page_warn_if_not_on_same_network);
-    var account = web3.eth.accounts[0];
+    waitforWeb3(function() {
```

Comment:
This seems like there a lot of changes, but it's just putting the code below inside the `waitForWeb3` callback. This was necessary because otherwise web3 was undefined. There's a bug here: if the user clicks the submit button in the payout page and web3 is not available yet, the button doesn't do anything. Maybe it should be disabled until web3 is available?

Reply:
Btw you can set GitHub's diff to ignore whitespace to check this file.